ansibullbot/utils/moduletools.py

```diff
@@ -53,6 +79,17 @@ def __init__(self, maintainers=None, gh_client=None):
         self.scraper_cache = os.path.expanduser(self.scraper_cache)
         self.gws = GithubWebScraper(cachedir=self.scraper_cache)
         self.gqlc = gh_client
+        self.files = []
+
+        # sqlalchemy
+        unc = 'sqlite:///'
+        unc += os.path.expanduser('~/.ansibullbot/cache/module_indexer.db')
```

Comment:
`--cachedir` value should be used for parent database directory. `ansible_module_indexer.db` could be used  instead of `module_indexer.db`.